daemon/entrypoint.sh

```diff
@@ -10,6 +10,7 @@ set -e
 : ${NETWORK_AUTO_DETECT:=0}
 : ${MDS_NAME:=mds-${HOSTNAME}}
 : ${OSD_FORCE_ZAP:=0}
+: ${OSD_PREPARE_FLAGS:=''}
```

Comment:
I would have a mild preference to name this something more related to what it actually does:  `CEPH_DISK_PREPARE_FLAGS`, perhaps.  It is verbose, but since it isn't really related to OSDs themselves (rather, the ceph-disk _way_ of manipulating OSDs), I'm uncomfortable calling it `OSD_PREPARE_FLAGS`, succinct though that is.